backend/models/core/Entity.js

```diff
@@ -109,6 +114,67 @@ class Entity {
           value_double
         })); 
   }
+
+  static async create({entries, columns, slug, entity_type_id}) {
+    const db = new DB();
+
+    let attributesValues = ``, insertAttributesSQL = ``;
+
+    let values =``, insertValuesSql = ``;
+    let dec = columns.length - 1;
+
+    const insertEntitiesSQL = `INSERT into entities (slug, entity_type_id) VALUES (:slug, :entity_type_id)`;
+    
+    const isEntities = await db.exectuteStatement(insertEntitiesSQL, [
+      {name: 'slug', value:{stringValue: slug}},
+      {name: 'entity_type_id', value:{longValue: entity_type_id}},
+    ]);
+    
+    if(isEntities) {
+      
+    let temp = [];
+
+    columns.map((col, i) => {
+      entries[`${col}_type`] === 'text' ? temp.push({value_string: entries[col]}) : null;
+      entries[`${col}_type`] === 'textarea' ? temp.push({value_long_string: entries[col]}) : null;
+      entries[`${col}_type`] === 'float' ? temp.push({value_double: entries[col]}) : null;
+
+      attributesValues += ` ${i > 0 ? ',' : ''}('${col}', (SELECT ID from entities ORDER by id desc limit 1),  '${entries[`${col}_type`]}', ${i+1})`;
+      values += ` ${i > 0 ? ',' : ''}((SELECT ID from entities ORDER by id desc limit 1), (SELECT ID from attributes ORDER by id desc limit 1)-${dec},${entries[`${col}_type`] === 'text' ? entries[col] : null}, ${entries[`${col}_type`] === 'textarea' ? entries[col] : null}, ${entries[`${col}_type`] === 'float' ? entries[col] : null})`;
```

Comment:
This assumes that the items that you just inserted would be the last entry in the table. This would fail if someone else is updating the database simultaneous with you.